src/features/transactions/swapEventSubscriber.ts

```diff
@@ -0,0 +1,144 @@
+import { Dispatch } from "@reduxjs/toolkit";
+
+import BigNumber from "bignumber.js";
+import { Contract } from "ethers";
+
+import { store } from "../../app/store";
+import { notifyTransaction } from "../../components/Toasts/ToastController";
+import { mineTransaction } from "./transactionActions";
+import {
+  LastLookTransaction,
+  SubmittedOrder,
+  SubmittedTransaction,
+  TransactionsState,
+} from "./transactionsSlice";
+
+const handleReceipt = ({
+  nonce,
+  transactionHash,
+  signerWallet,
+  transaction,
+  protocol,
+  chainId,
+  dispatch,
+}: {
+  nonce: string;
+  transactionHash: string;
+  signerWallet: string;
+  transaction: any;
+  protocol: "last-look" | "request-for-quote" | undefined;
```

Comment:
`protocol?: "last-look" | "request-for-quote"`

Reply:
Put the fix in #280 :)